packages/web-components/src/components/cbp-list/cbp-list.scss

```diff
@@ -0,0 +1,50 @@
+/**
+ * @prop --cbp-list-item-color: var(--cbp-color-text-darkest);
+ * @prop --cbp-list-item-color-dark: var(--cbp-color-text-lightest);
+*/
+
+:root {
+  --cbp-list-item-color: var(--cbp-color-text-darkest);
+  --cbp-list-item-color-dark: var(--cbp-color-text-lightest);
+
+  // --cbp-list-item-bullet: '-';
+}
+
+
+[data-cbp-theme=light] cbp-list[context*=dark]:not([context=light-always]),
+[data-cbp-theme=dark] cbp-list:not([context=dark-inverts]):not([context=light-always]) {
+  --cbp-list-item-color: var(--cbp-list-item-color-dark);
+}
+
+cbp-list {
+  line-height: var(--cbp-line-height-sm);
+
+  li {
+    color: var(--cbp-list-item-color);
+    font-size: var(--cbp-font-size-body);
+    font-weight: var(--cbp-font-weight-bold);
+    display: list-item;
+  }
+
+  //Unordered nested lists
+  li + ul{
```

Comment:
This is wrong. Nested lists need to be inside of list items. They cannot be siblings. Your nested selectors should look like `ul ol, ol ol {...}`